lib/routes.jsx

```diff
@@ -115,6 +115,8 @@ var routeElements = Object.keys(pages).map(function(path) {
   return <Route path={path} component={pages[path]} key={path}/>;
 });
 
+var makerPartyLocales = ['de', 'fr', 'es', 'pl', 'it', 'nl', 'cs', 'bg'];
```

Comment:
I'm not aware of the specific differences for French and Spanish, but if we're not talking about currency or units of measurement, I hardly notice the difference between en-US and en-GB. I'd guess that if these small pages are in the base language itself, that's good enough to get the message across such that no one's going to be insulted it's not localized in their region.